lib/ducer/ducer.h

```diff
@@ -107,7 +107,8 @@ namespace Ducers {
         data[i] = interpolateLow(_adcs[_adcIndices[i]]->readData(_adcChannels[i]));
         i++;
       } else {
-        data[i] = interpolateHigh(_adcs[_adcIndices[i]]->readData(_adcChannels[i]));
+        // data[i] = interpolateHigh(_adcs[_adcIndices[i]]->readData(_adcChannels[i]));
+        data[i] = _adcs[_adcIndices[i]]->readData(_adcChannels[i]);
```

Comment:
Was having issues with on-board interpolation for high pressure on 2/13/21. Nolan is looking into them, for now using the interpolation on the ground station side